src/axios.ts

```diff
@@ -1,11 +1,25 @@
 import axios, { AxiosRequestConfig, AxiosError, AxiosResponse } from 'axios';
+import http from 'http';
+import https from 'https';
 import emitter from './emitter';
 
+const agentOptions = {
+  keepAlive: true,
+  maxSockets: 100, // Maximum number of sockets to allow per host. Defaults to Infinity.
+  maxFreeSockets: 10,
+  // timeout: 60000, // active socket keepalive for 60 seconds
```

Comment:
`question`: should this be commented out?

Reply:
done